packages/shared/src/types/firestore.ts

```diff
@@ -425,6 +444,16 @@ export interface SlotSanityCheckReport {
   dateMismatches: Record<string, DateMismatchDoc>;
   attendanceFixes?: AttendanceAutofixReport;
 }
+export interface BookingsSanityCheckReport {
+  /** ISO timestamp of the sanity check run */
+  id: string;
+  /** A record of "unpaired" bookedSlots which have no equivalent slot with the same id (keyed by slot id)*/
+  missingSlots: Record<string, BookingsUnpairedCheckPayload>;
+  /** A record of bookedSlots for which the date is mismatched across collections (keyed by slot id) */
+  invalidDateBookings: Record<string, BookingsUnpairedCheckPayload>;
+  /** A record of bookedSlots for which the interval doesn't exist in respective slot (keyed by slot id) */
+  invalidIntervalBookings: Record<string, BookingsUnpairedCheckPayload>;
+}
```

Comment:
This is more less as I've envisioned it, only:
- I would rename `missingSlots` -> `strayBookings`
- I would rename `invalidDateBookings` -> `dateMismatches` and have each entry be `DateMismatchDoc` (conveying just enough information)
- for `invalidIntervalBookings` I would have each entry be:
```ts
{ 
  slotIntervals: Pick<SlotInterface, "intervals">,
  bookedInterval: Pick<CustomerBooking, "interval">
}
```